Sources/GRPC/AsyncAwaitSupport/GRPCAsyncServerCallContext.swift

```diff
@@ -83,26 +83,42 @@ public final class GRPCAsyncServerCallContext {
   @usableFromInline
   internal let userInfoRef: Ref<UserInfo>
 
-  /// Metadata to return at the end of the RPC. If this is required it should be updated before
-  /// the `responsePromise` or `statusPromise` is fulfilled.
-  public var trailers: HPACKHeaders {
+  /// Metadata to return at the start of the RPC.
+  ///
+  /// If this is required it should be updated before the first response is sent via the response
+  /// stream writer.
```

Comment:
I think we should state explicitly that the response headers are sent when the first response message is sent and anything set after that will be ignored.

Reply:
OK.

Reply:
Done in  c5cfcb1.